package.json needs `browser` key for client-side compatibility

In `package.json`:

```
…
"main": "lazy.node.js",
…
```

This results in the following error when using with Webpack:

```
Module not found: Error: Cannot resolve module 'fs' in ./node_modules/lazy.js
```

Using the [`browser` key](https://gist.github.com/defunctzombie/4339901) fixes this.

--- a/package.json
+++ b/package.json
@@ -8,6 +8,7 @@
     "email": "daniel.tao@gmail.com",
     "url": "http://philosopherdeveloper.com"
   },
+  "browser": "lazy.js",
   "main": "lazy.node.js",
   "license": "MIT",
   "repository": {

